cache/cache.go

```diff
@@ -53,40 +69,58 @@ func (c *Cache) Get(key string) (interface{}, bool) {
 		return nil, false
 	}
 
-	return object.Contents, true
+	return object.contents, true
 }
 
-// Objects returns all items in the cache as a map of objects, with a string as the key.
-func (c *Cache) Objects() map[string]Object {
+// Objects returns all objects in the cache.
+func (c *cacheImpl) Objects() (m map[string]interface{}) {
 	c.mutex.RLock()
 	defer c.mutex.RUnlock()
-	return c.objects
+
+	sz := len(c.objects)
+	if sz == 0 {
+		return
+	}
+
+	m = make(map[string]interface{}, sz)
+	for k, v := range c.objects {
+		m[k] = v.contents
+	}
+	return
 }
 
 // Purge removes ALL objects from the cache.
-func (c *Cache) Purge() {
+func (c *cacheImpl) Purge() {
 	c.mutex.Lock()
 	defer c.mutex.Unlock()
-	c.objects = map[string]Object{}
+	c.objects = map[string]object{}
 }
 
-// Set creates an object in the cache with a specified TTL `d` (in seconds). Note that TTL must be a
-// positive, non-zero integer. If the object already exists, it is overwritten. For bulk operations,
-// you may wish to use Supplant(map[string]cache.Object) in your code instead.
-func (c *Cache) Set(key string, val interface{}) {
+// Set creates an object in the cache. If the object already exists, it is overwritten.
+// For bulk operations, you may wish to use Supplant() instead to avoid the overhead of
```

Comment:
Um .. except that `Supplant` actually replaces the entire map, instead of just doing a bulk add; they're different operations.